### DISCUSSION TITLE
Any production shops using AlpineJS in their Shopify Themes?

### DISCUSSION THREAD

**Discussion by ruie** [alpinejs/alpine#1340] [4. General]
Hi guys,

I'm just wondering if somebody is using Alpinejs instead of jquery for their Shopify themes?

Thanks guys!

**Comment by vitalijalbu** [comment]
mee too searching for them

**Comment by vitalijalbu** [comment]
https://www.mahigold.com/

**Comment by ekwoka** [comment]
I work on quite a few, like the aforementioned Mahigold, and https://osoandme.com and https://lysbeauty.com among others.

**Reply by ghost** [reply]
> I work on quite a few, like the aforementioned Mahigold, and https://osoandme.com and https://lysbeauty.com among others.
Woow...

Basically I don't understand how to structure components to don't repeat code... 
For example **accordions, modals, dropdowns** etc...

**Reply by ekwoka** [reply]
You can just make a alpine component in your scripts for it.
